_docs/developer/coding_style_guide/php.md

```diff
@@ -0,0 +1,78 @@
+---
+title: PHP
+category: Developer
+order: 6
+---
+
+__Minimum Version__: 7.2
+
+For PHP, we use a foundation of [PSR-1](https://www.php-fig.org/psr/psr-1/)
+and [PSR-2](https://www.php-fig.org/psr/psr-2/) and then customize from
+there. These customizations are highlighted below:
+
+### Linting Code
+
+We use a custom standard for the [phpcs](https://github.com/squizlabs/PHP_CodeSniffer) tool,
+available at [Submitty/submitty-php-codesniffer](https://github.com/Submitty/submitty-php-codesniffer).
+You can run this against your code by running it (assuming in the `site/` directory):
+```
+vendor/bin/phpcs --standard=Submitty path/to/dir/or/file.php
+```
+
+### Classes, Methods
+
+* The opening brace for the class MUST go on the same line as the class name; the closing brace
+  for the class MUST go on the next line after the body.
+* The opening brace for a method MUST have one space between it and the closing parenthesis.
+* The closing brace for a method MUST go on the next line following the body.
+
+```
+class Test {
+    public function foo() {
+        // code
+    }
+
+    public function method(
```

Comment:
Will putting `$arg1`, `$arg2` etc in a new line be a recommended practice?
To me, it generally looks better to have them in one line if they are short.

Like this:
`public function method($arg1, $arg2)`

Reply:
It's more just to showcase how multilinear argument lists might work if you have a bunch of variables or long names that would push you over the 120 character limit